tools/feed-cleaner/Clean-Feeds.ps1

```diff
@@ -0,0 +1,67 @@
+param(
+    [Parameter(Mandatory=$true)][string]$Organization,
+    [Parameter(Mandatory=$true)][string]$FeedProject,
+    [Parameter(Mandatory=$true)][string]$RunProject,
+    [Parameter(Mandatory=$true)][string]$AccessToken    
+)
+
+$ErrorActionPreference = "$Stop"
+
+function Get-Feeds([string]$Organization, [string]$Project, [PSCredential]$Credential) {
+    $url = "https://feeds.dev.azure.com/$Organization/$Project/_apis/packaging/feeds?api-version=4.1-preview"
+    
+    Write-Information "Requesting list of feeds from: $url"
+    $response = Invoke-RestMethod -Uri $url -Method GET -Credential $Credential -Authentication Basic
+
+    Write-Information "Found $($response.value.Length) feeds."
+
+    return $response.value
+}
+
+function Get-FilteredFeeds([string]$Organization, [string]$Project, [PSCredential]$Credential) {
+    $feeds = Get-Feeds -Organization $Organization -Project $Project -Credential $Credential
+
+    # Filter the list of feeds to those in the form azure-sdk-{runid} as those are the ones
+    # that we want to evaluate for deletion.
+    Write-Information "Filtering down to burner feeds with the form azure-sdk-{runid}."
+    $filteredFeeds = @($feeds | Where-Object { $_.name -match "^azure-sdk-([0-9]+)$" })
```

Comment:
Shouldn't this also use the expressionToExtractRunID you defined below?

Reply:
Done.